x-pack/test/kerberos_api_integration/fixtures/README.md

```diff
@@ -0,0 +1,10 @@
+# Kerberos Fixtures
+
+Kerberos fixtures are created with the following principals:
+
+* tester@TEST.ELASTIC.CO (password is `changeme`)
+* host/kerberos.test.elastic.co@TEST.ELASTIC.CO
+* HTTP/localhost@TEST.ELASTIC.CO
+
+The SPNEGO token used in tests is generated for for `tester@TEST.ELASTIC.CO`. We can re-use it multiple times because we
```

Comment:
Hey @bizybot ,

Can you please help us to understand whether the approach we're taking here for integration tests is going to work with Elasticsearch Kerberos support in the long run and we don't make any wrong assumptions?

The key here is that we disable replay cache (`-Dsun.security.krb5.rcache=none`) so that we can use the same SPNEGO token without any interval between calls and use very large `clockskew` (2147483647 seconds or ~68 years) to use the same token for a very long time. It looks like in this case we don't need to have a dedicated KDC for tests, unless we're missing something.

Thanks!

Reply:
And the second question, it seems we can't exchange SPNEGO token to an access token for a "normal" user using this:
```
POST /_security/oauth2/token 
{
  "grant_type" : "client_credentials"
}
```
I'm getting `action [cluster:admin/xpack/security/token/create] is unauthorized for user ... `. I thought we were supposed to call this API endpoint using `Negotiate xxx-spnego-token` and user won't need any admin privileges to get the token?

Reply:
While we discuss this in https://github.com/elastic/elasticsearch/issues/41943, it made more sense to point this out here: 

Using the `client_credentials` grant will get you an access token only, and no refresh token. This access token would only be usable for (usually 20min) max 1 hour depending on the configuration.

Reply:
Yep, thanks for the note! We're going to return `WWW-Authenticate: Negotiate` when access token expires to request a new SPNEGO token that we can exchange for a new access token.

Reply:
Thanks for looking into this @bizybot ! The SPNEGO token I've generated ~5 days ago is still accepted by the ES in integration tests, so I assume we should be fine, but would be great if you can confirm or refute that.

Reply:
Awesome, thanks!